### PR TITLE
CMS: adds contextualization script, fixes for VMs

### LINKED ISSUE #2044: Subtitles in the 2010 and 2011 VM records (252 250) and updates to 2011 VM record
- [ ] Change subtitles in VM records 252 and 250:

- How were these data selected? -> How was this built?
- How were these data validated? -> How to validate?
- How can you use these data? -> How to use?

Then for 2011 (252):

- [x] in description: "to be used with the 2011 CMS open data" -> "to be used with the 2011 and 2012 CMS open data"

- [x] add section subtitle "Notes" with "CMS-Open-Data-1.2.0.ova has a 20G virtual hard disk and a 10G cvmfs cache. CMS-Open-Data-1.3.0.ova has a 40G virtual hard disk and a 20G cvmfs cache. Use CMS-Open-Data-1.3.0.ova if you access the condition data for full event range for 2012 data (see `<a href=\"/docs/cms-guide-for-condition-database\">`the CMS guide to condition database`</a>` for further details."

- [ ] attach the file for CMS-Open-Data-1.3.0.ova from  https://cernbox.cern.ch/index.php/s/SkLCf42lo6DqdAE #1959

- [x] add a link to a new contextualization script, see issue #2045

### LINKED ISSUE #2045: Add a record for the contextualization script of the new VM version
File
[cms-user-data.txt](https://github.com/cernopendata/opendata.cern.ch/files/1538273/cms-user-data.txt)
and 
[cernvm-script.txt](https://github.com/cernopendata/opendata.cern.ch/files/1542054/cernvm-script.txt)

Checked with @jblomer: opendata-desktop-setting script is the the same as before

```
{
    "files": [
        {
            "checksum": "FIXME", 
            "uri": "FIXME", 
            "size": FIXME
        }, 
        {
            "checksum": "FIXME", 
            "uri": "FIXME", 
            "size": FIXME
        }, 
        {
            "checksum": "sha1:3248e8702168d4f3389eee14e31df330e27089ac", 
            "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2011/opendata-desktop-settings", 
            "size": 153
        }
    ], 
    "accelerator": "CERN-LHC", 
    "abstract": {
        "description": "The scripts provided create the CMS virtual machine image based on <a href=\"https://cernvm.cern.ch\">CernVM</a>, with Scientific Linux CERN 6 (slc6)."
    }, 
    "run_period": "Run2012A-Run2012D", 
    "recid": "251", 
    "date_published": "2017", 
    "authors": [
        {
            "name": "Blomer, Jakob"
        }
    ], 
    "publisher": "CERN Open Data Portal", 
    "title": "Contextualisation scripts to create the CMS VM Image version 1.3.0, for 2012 data release", 
    "control_number": "FIXME", 
    "relations": [
        {
            "type": "isRelatedTo", 
            "recid": "252", 
            "description": "The final CERN VM image can be found here:"
        }
    ], 
    "experiment": "CMS", 
    "collections": [
        "CMS-Tools"
    ], 
    "usage": {
        "description": " <p> First, you need to run a normal Micro-CernVM, using your own context so you can login. Copy the attached files to the running VM and then   login.</p>\n <p>Inside the VM, you first need a copy of the latest Micro-CernVM release as a template for the Open Data image. Download it, inside the running VM, from: <a href=\"http://cernvm.cern.ch/releases/production/cernvm3-micro-2.8-6.hdd\">    http://cernvm.cern.ch/releases/production/cernvm3-micro-2.8-6.hdd </a>  placing it in the same directory as the other files. </p>\n <p> Run the script:<code>./cernvm-script </code>which will produce some output on the terminal, and then finish. </p>\n      <p>The result should be a file with the title coming from the -n field in the script. In this case, the file we're interested in is: <code>CMS-OpenData-1.3.0.ova </code>which can be copied from the running VM to elsewhere. </p> "
    }, 
    "date_created": "2012", 
    "distribution": {
        "formats": [
            "txt"
        ]
    }, 
    "type": {
        "primary": "Environment", 
        "secondary": [
            "VM"
        ]
    }
}

```

In addition, please correct and update the following in all contextualization script records (249, 251 and this new one)

- [x] remove  
```
 "system_details": {
        "release": "Use this with a Micro-CernVM available at", 
        "url": "http://cernvm.cern.ch/portal/downloads"
    }, 
```
as it appears with the irrelevant text "Recommended release for analysis:" on the portal

- [x] update "abstract" in 251 with
```
"abstract": {
        "description": "The scripts provided create the CMS virtual machine image based on <a href=\"https://cernvm.cern.ch\">CernVM</a>, with Scientific Linux CERN 6 (slc6)."
    }, 
```
and in 249 with
```
"abstract": {
        "description": "The scripts provided create the CMS virtual machine image based on <a href=\"https://cernvm.cern.ch\">CernVM</a>, with Scientific Linux CERN 6 (slc5)."
    }, 
```

- [ ] Replace the subheader for "usage" "How can you use these data?" with "How can you use this?"

### PR DESCRIPTION
**Pull request by ArtemisLav**
* Adds one record for the new contextualization script and makes
modifications to the previous contextualization script records. (closes #2045)

* Updates the 2011 VM record, so that it is appropriate for 2012 data as well. (closes #2044)

Signed-off-by: Artemis Lavasa <artemis.lavasa@cern.ch>

### CODE REVIEW

**Comment by ArtemisLav** [comment]
Both those issues required changes to the same files, so I have addressed them on the same PR. 
I just need to update this with the file information (two files for #2045 and one file for #2044).

File: cernopendata/modules/fixtures/data/records/cms-tools-vm-image-2012.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "date_published": "2017", 
+  "distribution": {
+      "formats": [
+          "txt"
+      ],
+      "number_files": 3,
+      "size": "FIXME"
+  }, 
+  "experiment": "CMS", 
+  "files": [
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-tools-vm-image-2012.json]: Here is the file information:
```
{
  "files": [
    {
      "checksum": "adler32:234a36f3",
      "size": 178,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/cernvm-script.txt"
    },
    {
      "checksum": "adler32:f223d6d8",
      "size": 3071,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/cms-user-data.txt"
    },
    {
      "checksum": "adler32:786f33c7",
      "size": 152,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/opendata-desktop-settings"
    }
  ]
}
```

Note that I create 2012 directory, as we don't have versions in 2010 one...

File: cernopendata/modules/fixtures/data/records/cms-tools-vm-image-Run2011A.json [hunk 2]
```diff
@@ -16,21 +16,31 @@
...
+    "number_files": 2,
+    "size": "FIXME"
   }, 
   "experiment": "CMS", 
   "files": [
     {
       "checksum": "sha1:7ea617ce250cb3794c6e3b57a0eff38c15fa5938", 
       "size": 18145280, 
       "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2011/CMS-Open-Data-1.2.0.ova"
+    },
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-tools-vm-image-Run2011A.json]:
```
    {
      "checksum": "adler32:4a0b475e",
      "size": 18728960,
      "uri": "root://eospublic.cern.ch//eos/opendata/cms/environment/2012/CMS-Open-Data-1.3.0.ova"
    },

```

Comment (ArtemisLav) [reply cernopendata/modules/fixtures/data/records/cms-tools-vm-image-Run2011A.json]: Thanks, I have added everything.

File: cernopendata/modules/fixtures/data/records/cms-tools-vm-image.json [hunk 3]
```diff
@@ -91,8 +93,10 @@
   "distribution": {
     "formats": [
       "txt"
-    ]
-  }, 
+    ],
+    "number_files": 3,
+    "size": "4091"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-tools-vm-image.json]: `"size": 4091` it must be an integer

**Pull request merged**